MyCashflow/Default-Prints-Theme
# Add default print labels

Added default print labels:
* product-barcode-sticker-40x39mm (small sticker with barcode to be attached to the product)
* product-sheet-A4 (product information sheet with image, includes qr-code with link to product)
* storage-shelf-label-90x38mm (for warehouse picking, includes barcode)
* store-shelf-label-90x38mm (store shelf label, includes qr-code and barcode)

## How to test

Use the "Print product labels"-feature to easily print out information sheets or store/storage labels about your products.

<img width="900" alt="Screenshot 2024-09-13 at 7 19 23" src="https://github.com/user-attachments/assets/570427ba-3c15-4c5e-9b3a-bc8cd3250b8b">

Check that the information is printed on the labels without any issues. Below are the visual presentations and required information for each. For testing try out products with varying amount of information.

### Store shelf label
Size: 90mm x 38mm
Product information:

- Product name (max. 2 lines)
- Variation name
- Product number
- Price / Campaign price and regular price + 30-day price
- Barcode and QR code

![Tulostettava myymälän hintalappu V2](https://github.com/user-attachments/assets/42b533dd-1c3f-4c33-b818-478824057ee1)

### Storage shelf label
Size: 90mm x 38mm
Product information:

- Product name
- Variation name
- Product number
- Storage location
- Barcode and QR code

![Tulostettava varaston hintalappu_versio2](https://github.com/user-attachments/assets/311add65-87dc-45ba-a0d4-054e9d2a1a40)

### Product sheet A4
Size: A4
Product information:

- Product name
- Variation name
- Price
- Campaign price
- Campaign validity period
- Main product image
- Product categories
- Brand
- Product number
- Description
- Product Features
- QR code

### Mini product sticker
Size: 40mm x 39mm
Product information:  

- Product name  
- Variation name  
- Product number  
- Barcode

## Notices

Product sheet A4 requires few translations and these will be updated later after they are available.
